src/components/PullRequestsTable/PullRequestsTable.tsx

```diff
@@ -14,14 +14,60 @@
  * limitations under the License.
  */
 import React, { FC, useState } from 'react';
-import { Typography, Box, ButtonGroup, Button } from '@material-ui/core';
+import { Typography, Box, ButtonGroup, Button, makeStyles } from '@material-ui/core';
 import GitHubIcon from '@material-ui/icons/GitHub';
 import { Table, TableColumn } from '@backstage/core';
 import { useProjectName } from '../useProjectName';
 import { usePullRequests, PullRequest } from '../usePullRequests';
 import { PullRequestState } from '../../types';
 import { Entity } from '@backstage/catalog-model';
 
+const useStyles = makeStyles(() => ({
+  status: {
+    fontWeight: 500,
+    '&::before': {
+      width: '0.7em',
+      height: '0.7em',
+      display: 'inline-block',
+      marginRight: 8,
+      borderRadius: '50%',
+      content: '""',
+    },
+  },
+  open: {
+    '&::before': {
+      backgroundColor: '##28A745',
+    },
+  },
+  closed: {
+    '&::before': {
+      backgroundColor: '#6F42C1',
```

Comment:
Red would be a better color for closed. Purple indicates merged, a more specific form of closed.